docs/source/tutorial/typesfuns.rst

```diff
@@ -1174,7 +1172,7 @@ name. For example, a pair type could be defined as follows:
         snd : b
 
 Using the ``Class`` record from earlier, the size of the class can be
-restricted using a ``Vect`` and the size included in the type by parameterising
+restricted using a ``Vect``, and the size included in the type by parameterising
```

Comment:
I believe that comma should be after "type"? The class can be restricted by two things, which should be grouped and not comma-separated, with the second part of the sentence then clearly starting by having the comma after "type"?